core/src/main/resources/hudson/model/Computer/index.jelly

```diff
@@ -28,6 +28,21 @@ THE SOFTWARE.
     <st:include page="sidepanel.jelly" />
     <l:main-panel>
 
+      <!-- Node name and status -->
+      <h1>
+        <span class="icon-md">
+          <l:icon src="${it.iconClassName}" />
+        </span>
```

Comment:
```suggestion
        <l:icon src="${it.iconClassName}" class="icon-xlg" />
```

Reply:
I has try the solution too, but I think there was no space between icon and text. I will retry it little later, because I am compiling other branch now. And are you sure 'xlg'? That means extra-large, right?

Reply:
here is what I have locally with my suggestion

![Screen Shot 2022-06-10 at 18 03 09](https://user-images.githubusercontent.com/985955/173106161-67f65b85-b39f-47f5-a8d7-41cb53548657.png)

Reply:
@alecharp  So I do quick tests
```
<h1><l:icon src="${it.iconClassName}" class="icon-xlg" />${it.caption}</h1>
<h1><l:icon src="${it.iconClassName}" class="icon-lg" />${it.caption}</h1>
<h1><l:icon src="${it.iconClassName}" class="icon-md" />${it.caption}</h1>
<h1><l:icon src="${it.iconClassName}" class="icon-sm" />${it.caption}</h1>
```
 and it looks like this
![image](https://user-images.githubusercontent.com/89339813/173106229-1d2b5db9-297f-4720-bb0b-ac2114a036a0.png)

 Which one you find the best? I do not like any of them (do not ask why? just it looks terrible for me ;-))

Reply:
thx. I see. Nice article. You true, I do not like it too. It is more or less trash. But in this case it is necessary, because it represent node online / offline state. Otherwise the whole page shall be reworked as well. An I want only change the description. (general I do not understand, why no body change the pages about nodes, labels, node_minotoring ... . It is
one of the most important part in Jenkins, but I will do my best to make the pages more modern and fit with jenkins design-library fit - of cores in other PR :-)) again thx for review.

Reply:
Yes absolutely not suggesting a change here :)